test/integ_test/aml_model_test/conda.yaml

```diff
@@ -11,6 +11,6 @@ dependencies:
       - onnxruntime
       - datasets
       - scipy
-      - transformers
+      - transformers==4.31.0 # TODO(team): Fixed error and update to latest version
```

Comment:
Could you add some details about what the error is?

Reply:
I created a task for this with error details and added the task id here.

Reply:
Thanks!